src/httpServer/handler.ts

```diff
@@ -56,6 +56,7 @@ export interface HandlerContext extends DBOSContext {
   retrieveWorkflow<R>(workflowUUID: string): WorkflowHandle<R>;
   send<T>(destinationUUID: string, message: T, topic?: string, idempotencyKey?: string): Promise<void>;
   getEvent<T>(workflowUUID: string, key: string, timeoutSeconds?: number): Promise<T | null>;
+  getWorkflows(input: GetWorkflowsInput): Promise<GetWorkflowsOutput>;
```

Comment:
How about rename it to `listWorkflows` or something that indicates it's a query?

Reply:
The current name indicates that well I think